master-firmware/src/strategy.cpp

```diff
@@ -168,53 +105,23 @@ static void wait_for_autoposition_signal(void)
     wait_for_color_selection();
 }
 
-bool strategy_puck_is_picked(manipulator_side_t side)
-{
-    bool res = true;
-    if (USE_LEFT(side)) {
-        res &= fabsf(motor_get_current("left-pump-1")) > config_get_scalar("master/arms/left/gripper/current_thres")
-            && fabsf(motor_get_current("left-pump-2")) > config_get_scalar("master/arms/left/gripper/current_thres");
-    }
-    if (USE_RIGHT(side)) {
-        res &= fabsf(motor_get_current("right-pump-1")) > config_get_scalar("master/arms/right/gripper/current_thres")
-            && fabsf(motor_get_current("right-pump-2")) > config_get_scalar("master/arms/right/gripper/current_thres");
-    }
-    return res;
-}
-
-void strategy_shutdown_endgame(void)
-{
-    manipulator_gripper_set(BOTH, GRIPPER_OFF);
-}
-
-void strategy_order_play_game(strategy_context_t* ctx, RobotState& state)
+void strategy_order_play_game(RobotState& state, enum strat_color_t color)
```

Comment:
pointers are more in your face IMHO

Reply:
Not sure I understand the comment here

Reply:
I would pass `RobotState` as a pointer instead of a reference

Reply:
But then it could be null #NoFreeLunch 😭

Reply:
Yes I think I will keep as non nullable for now and merge it like this.

Reply:
:shipit: